docs/source/tools/assistant.rst

```diff
@@ -132,6 +133,8 @@ Here is what each field means:
   - ``grpc-timeout``: This option controls the timeout used for communicating
     with the scenario service. If unspecified this defaults to 60s. Unless you get
     errors, there should be no reason to modify this.
+  - ``jvm-options``: A list of options passed to the JVM when starting the scenario
+    service. This can be used to limit maximum heap size via ``-Xmx500M``.
```

Comment:
I think we can probably avoid giving an example. here. If you don't know how this flag works, you should not be using it.

```suggestion
    service. This can be used to limit maximum heap size via the ``-Xmx`` flag.
```